lib/types/time-uuid.js

```diff
@@ -378,8 +378,8 @@ function getOrGenerateRandom(id, length, callback) {
  * @private
  * @param {Date} date
  * @param {Number} ticks
- * @param {String|Buffer} nodeId
- * @param {String|Buffer} clockId
+ * @param {string|Buffer} nodeId
```

Comment:
if we are writing `string` in lower case then we should start writning `number` in lower case for consistency